TreeTracker/Resources/Localizable.strings

```diff
@@ -99,3 +100,25 @@
 "Settings.PhotoQuality.Option.Medium.Info" = "Medium resolution, medium data usage.";
 "Settings.PhotoQuality.Option.High.Title" = "High";
 "Settings.PhotoQuality.Option.High.Info" = "High resolution, high data usage.";
+
+//ChatList Screen
+"ChatList.SyncMessages.normal.Title" = "Sync Messages";
+"ChatList.SyncMessages.loading.Title" = "Loading...";
+"ChatList.LastSyncLabel.WithouDate.Text" = "Please, tap the button above to sync messages.";
+"ChatList.LastSyncLabel.WithDate.Text" = "Last sync time:";
+"ChatList.NoMessagesLabel.Text" = "No messages";
+"ChatList.MessageChatCell.FallbackTitle" = "Admin";
+"ChatList.DefaultCell.FallbackTitle" = "";
+
+//Messages Screen
+"Messages.Title" = "Admin";
+"Messages.InputTextView.PlaceHolder" = "Click to write a message";
```

Comment:
I always felt like click is a web term🤷🏿‍♂️. Personal preference would be `Start conversation`, `send a message` or `new message`